File: Easy/1323-maximum-69-number/1323-maximum-69-number.java
class Solution {
    public int maximum69Number (int num) {
        String str = String.valueOf(num);
        StringBuilder sb = new StringBuilder();
        boolean isChanged = false;
        for(char ch : str.toCharArray())
        {
            if(ch == '6' && !isChanged)
            {
                ch = '9';
                isChanged = true;
            }
            sb.append(ch);
        }
        
        return Integer.parseInt(sb.toString());
    }
}

File: Easy/1323-maximum-69-number/1323-maximum-69-number.py
class Solution(object):
    def maximum69Number (self, num):
        word = str(num)
        answer = ""
        isChanged = False
        for i in word:
            if i == '6' and not isChanged:
                i = '9'
                isChanged = True
            
            answer = answer + i
        
        return int(answer)